### PR TITLE
Welcome Page Updates

### PR DESCRIPTION
**Pull request by heymonicakay**
### Changes Made

Modified README.md
- Cleaned up grammar
- Corrected typos
- Fixed spelling errors
- Changed training session language
- Updated session info re: virtual vs in-person
- Switched sessions from Thursdays to Mondays

### Steps To Review

1. Checkout my branch and view the changes.
```
git checkout -b mp-update-prework-sesssion-info
```
2. Proofread for spelling or grammatical mistakes.
3. Review the Foundations Sessions section and confirm the wording makes sense.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,31 +2,32 @@
...
 
-We want to ensure that you are as prepared as possible for the technologies and concepts that your instruction staff will be coaching you through in the first few weeks, so we have prepared this couse to set you up for success. These resources will provide you with the basic skills, terminology, and context for what it will be like at Nashville Software School.
+We want to ensure that you are as prepared as possible for the technologies and concepts that your instruction staff will be coaching you through in the first few weeks, so we have prepared this course to set you up for success. These resources will provide you with the basic skills, terminology, and context for what it will be like at Nashville Software School.
 
 ## Foundations Course is Mandatory
 
-If you don't do the **Foundations Course**, you will be immediately behind your teammates on the first day of class and are at risk of needing to move to the next cohort so that you have time to complete it.
+If you do not do the **Foundations Course**, you will be immediately behind your teammates on the first day of class and are at risk of needing to move to the next cohort so that you have time to complete it.
```
Comment (Scott47) [inline README.md:9]: I am struggling with this language. Do you know if there is a reason we say "behind your teammates". We spend so much time trying to keep them from comparing themselves to others. What if it said `If you do not complete the **Foundations Course**, you will be unprepared on the first day of class. Without this base of knowledge before you begin, you will be at risk of need to move to the following cohort pending your completion.` ?

File: README.md [hunk 2]
```diff
@@ -2,31 +2,32 @@
...
 
-Nashville Software School is an accelerated bootcamp, and you will hit the ground running on day one. We will assume that you've completed this work, and provide minimal resources to you if you have not. Additionally, the life of a software developer is one of focus and meeting committments. Completing the **Foundations Course** shows us that you have the ability to prioritize this work, can focus on it, and meet the committment of completing it.
+Nashville Software School is an accelerated Bootcamp, and you will hit the ground running on day one. We will assume that you have completed this work and provide minimal resources if you have not. Additionally, the life of a software developer is one of focus and meeting commitments. Completing the **Foundations Course** shows us that you can prioritize this work, focus on it, and meet the commitment of completing it.it.
```
Comment (Scott47) [inline README.md]: `Bootcamp` should be changed to `boot camp`. Also, you have an extra `it.` at the end of line 11.

File: README.md [hunk 3]
```diff
@@ -2,31 +2,32 @@
...
 
 ## Foundation Sessions
 
-Another opportunity we provide for your foundations are training sessions before your cohort starts. Starting 4 weeks before the beginning of the cohort, on each Thursday evening, you can come in to ask questions, meet your teammates, and get some mentoring on this **Foundations Course**. These are a great opportunity for you to make sure you are set up for success on day one of your cohort.
+Another opportunity we provide for your Foundations is coaching sessions every Monday evening. Starting approximately four weeks before your cohort begins, you can come to ask questions, meet your teammates, and get some mentoring from NSS Alumni and Junior Instructors on this **Foundations Course**.
```
Comment (Scott47) [inline README.md]: I think `Foundations` should be lowercase here. It is not referring to the title of the course, just an individual's foundational skills. 

I think that we may change the days/times of sessions depending on what is best. Can we say `Another opportunity we provide to reinforce your software development foundations is coaching sessions that start approximately four weeks before your cohort begins. You can come to ask questions, meet your teammates, and get some mentoring from NSS alumni and junior instructors.`

File: README.md [hunk 4]
```diff
@@ -2,31 +2,32 @@
...
 
-They start at 5:00 PM and go to 7:00 PM in one of the classrooms at NSS.
+The sessions are from 5:00 PM  to 7:00 PM.  They are an excellent opportunity for you to make sure you set yourself up for success on day one of your cohort.
```
Comment (Scott47) [inline README.md]: There is an extra space between `PM` and `to`. 

Next sentence should read `They are excellent opportunities...`

**Review by Scott47 — CHANGES_REQUESTED** [review]
This looks great. I left some comments with suggested changes. Thank you so much for tackling this ticket!

File: README.md (hunk 3 above)
Comment (heymonicakay) [reply README.md]: Thanks! I worked this into my changes. I agree it does sound better and it leaves room for us to make changes to the schedule when we need.

File: README.md (hunk 4 above)
Comment (heymonicakay) [reply README.md]: Whoops! I guess this is why we proofread haha thanks for catching this!

File: README.md (hunk 2 above)
Comment (heymonicakay) [reply README.md]: Got it!it!  :sweat_smile:

File: README.md (hunk 1 above)
Comment (heymonicakay) [reply README.md:9]: Hmm, I'm not sure what the reasoning was behind that particular phrasing, but I do see what you are saying. It does feel kind of 'peer-pressure-y' in a way that, as you said, sort of contradicts the safe and welcoming learning environment of NSS. Pushing up changes in just a few!

**Comment by heymonicakay** [comment]
@Scott47 changes made! Thanks for your feedback, it was super helpful.

**Review by Scott47 — APPROVED** [review]
👍 Looks great! Thank you, @heymonicakay !

Editing this to see if it shows up in our channel.

**Comment by Scott47** [comment]
Will this comment show up in our slack channel?

File: README.md (hunk 1 above)
Comment (heymonicakay) [reply README.md:9]: @Scott47 I got some clarification around the decision to use the original phrasing, and the original wording _is_ intentional, so I'm pushing up changes to switch it back.

Comment (Scott47) [reply README.md:9]: 👍

**Review by Scott47 — APPROVED** [review]
👍 Ready for merge! 😄

**Pull request merged**